src/azure-cli/azure/cli/command_modules/acr/scope_map.py

```diff
@@ -14,12 +15,11 @@ def acr_scope_map_create(cmd,
                          resource_group_name=None,
                          description=None):
 
+    resource_group_name = get_resource_group_name_by_registry_name(cmd.cli_ctx, registry_name, resource_group_name)
     validate_premium_registry(cmd, registry_name, resource_group_name)
```

Comment:
`validate_premium_registry` internally fetches the `resource_group_name` if it's none. 

Since the method `get_resource_group_name_by_registry_name` already fetches the resource group, this can be used with `validate_premium_registry` to avoid an extra call to retrieve the resource group.